app.js

```diff
@@ -217,5 +222,57 @@ messageAllStreaksForChannel = channel => {
 }
 
 messageAllActiveStreaks = msg => {
-  msg.reply(`here are all the active streaks:\n` + bot.buildActiveStreaksMessage())
+  msg.reply(`here are all the active streaks:\n` + bot.buildActiveStreaksMessage(msg.guild.id))
+}
+
+canManageRoles = member => {
+  return member.permissions.bitfield & 0x00000008
```

Comment:
Can you add a comment explaining this as it's a bit funky

Reply:
sure, we should also look into if there is a more idiomatic way of doing this.